e2e/vm/additional_disk_test.go

```diff
@@ -38,6 +49,10 @@ var testAdditionalDisk = func(o *option.Option) {
 
 			newPsOutput := command.StdoutStr(o, "ps", "--all", "--format", "{{.Names}}")
 			gomega.Expect(newPsOutput).Should(gomega.Equal(oldPsOutput))
+
+			networks := command.StdoutStr(o, "network", "ls")
+			gomega.Expect(networks).Should(gomega.ContainSubstring(userNetwork))
+			gomega.Expect(command.StdoutStr(o, "start", "--attach", containerName)).Should(gomega.Equal(userFile))
```

Comment:
Checking for the file will always succeed because we `touch` the file as a part of the command, right? So this check doesn't actually ensure the file is persisted across reboots.

Perhaps we can start the container with e.g.
```
finch run -d --name usertest -v testvolume:/myvolume public.ecr.aws/docker/library/alpine:latest sh -c "sleep infinity"
```
and then `finch exec` to do the other commands?